packages/backend/src/lib/Ethereum/Ethereum.ts

```diff
@@ -4,13 +4,12 @@ import { EthereumConfigDeployContract } from './EthereumConfig/EthereumConfigDep
 import { EthereumConfigMintNFT } from './EthereumConfig/EthereumConfigMintNFT';
 import { EthereumConfigReadSmartContract } from './EthereumConfig/EthereumConfigReadSmartContract';
 import { EthereumConfigReadUserDataFromSmartContract } from './EthereumConfig/EthereumConfigReadUserDataFromSmartContract';
-import { readFileSync, writeFileSync } from 'fs';
```

Comment:
don't create unnecessary merge conflicts with other branches by changing unreleated stuff. Leave this to the person that works on this code.